### DISCUSSION TITLE
Easily visible "What's New (snatched/watched)" feature

### DISCUSSION THREAD

**Discussion by rafi-d** [pymedusa/Medusa#7233] [Ideas & Feature requests]
The problem: it is no possible to easily see what new episodes have been snatched since you last used Medusa (or you have not watched yet... )

Solution: Add a new column to the main screen: "New/Not-Watched". It will have a bright colored counter, which will increment for every newly snatched  episode. The color will change (or font from bold to regular) when medusa starts.

The user can: 
A. decrement one show-counter when he watched an episode, or reset it to 0
B. reset the whole column to 0
C. have an option to auto-reset a show/column counter to 0 every time Medusa starts (per show or globally)

With this, the user can simply and very easily  see what has been snatched when he comes back home from work/school, and his "to-watch" list...

Another helpful option: persistent notifications  for new snatches (this can improves only option C).
Yet one more option: read and reset a counter from the "watched" status from Kodi.

I believe this quick status check can be very helpful.

Thanks

**Comment by p0psicles** [comment]
What about the history page?

What about using notifications? I use telegram. Have created a dedicated group that always shows me latest snatch/downloads.

Soon we'll have added the possibility to change status watched for an episode. Integration with Kodi, Plex and trakt will follow. We could create a page from that.

**Comment by rafi-d** [comment]
I don't like the history page, which is kind of a log, and not per-show. Also does not highlight new from last run.
Yes, you can "invest" and use your own notifications, but the simple user relies on Medusa to provide the info, in a way it can be easily seen. 

I am speaking of some *accumulated* view, at least per-show, maybe even total, and the natural place/page to do that (as of now) is the main page. I guess this page will still be there, forever...

I guess you can click on any counter so to jump to another page (log/history/watched)  too...
